### DISCUSSION TITLE
Prevent user change their password?

### DISCUSSION THREAD

**Discussion by orchardc** [OrchardCMS/OrchardCore#12546] [Q&A]
We have a need to prevent user change their password.

![image](https://user-images.githubusercontent.com/16193516/193462124-890fa5c8-83a3-490f-98a9-f3061b2da291.png)
This option is cleared but the user still be able to see and change their password - is it a bug?

![image](https://user-images.githubusercontent.com/16193516/193462048-3fe1f38b-7067-4a05-9201-cccba7aa2f47.png)

**Comment by hishamco** [comment]
We might add a feature similar to `UsersChange Email`

https://github.com/OrchardCMS/OrchardCore/blob/339d74b251b53393dd510f8ad3027aa17d78500d/src/OrchardCore.Modules/OrchardCore.Users/Manifest.cs#L17-L23
